src/mlpack/methods/preprocess/preprocess_imputer_main.cpp

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file preprocess_imputer_main.cpp
+ * @author Keon Kim
+ *
+ * a utility that provides imputation strategies fore
+ * missing values.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/core/data/imputer.hpp>
+#include <mlpack/core/data/dataset_info.hpp>
+#include <mlpack/core/data/map_policies/increment_policy.hpp>
+#include <mlpack/core/data/map_policies/missing_policy.hpp>
+#include <mlpack/core/data/imputation_methods/mean_imputation.hpp>
+#include <mlpack/core/data/imputation_methods/median_imputation.hpp>
+#include <mlpack/core/data/imputation_methods/custom_imputation.hpp>
+#include <mlpack/core/data/imputation_methods/listwise_deletion.hpp>
+
+PROGRAM_INFO("Impute Data", "This utility takes a dataset and converts user "
+    "defined missing variable to another to provide more meaningful analysis "
+    "\n\n"
+    "The program does not modify the original file, but instead makes a "
+    "separate file to save the output data; The program requires you to "
+    "specify the file name with --output_file (-o)."
```

Comment:
It says here that --output_file is required, but in the code it isn't required and only a warning is issued if --output_file is not specified. I think issuing a warning is a better approach, so maybe we should update the documentation here?